### DISCUSSION TITLE
[Feature Request] Enable Countdowns to handle larger numbers

### DISCUSSION THREAD

**Discussion by rishavs** [saadeghi/daisyui#2698] [Ideas / feature requests]
Hi
I love the countdown animation and want to use it for things like "likes" and comments count in my app.
Something similar to what Youtube does.

Currently, it seems that countdown only works for 2 digits. 
Would love for countdown to handle arbitrarily sized numbers so that we can use this effect for more types of use cases.

**Comment by saadeghi** [comment]
It's designed to show a countdown as for time, not a "counter".  
It can't be infinite because the way it works, needs a fixed width, so let's say if it's supposed to work with 4 digit numbers, `1` would show as `0001` which won't look good.

A workaround is to separate the number using JS and show the first 2 digits in `<span class="countdown">`

Let me know if you have a question.
